packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -37,18 +37,21 @@ import {
 } from '../test_data/policies'
 import { createPolicyAndGeographyFactory, PolicyFactory, RulesFactory } from './helpers'
 
+const mockStream = stream.mockStream(PolicyStreamKafka)
 const GeographyServer = GeographyServiceManager.controller()
 const PolicyServer = PolicyServiceManager.controller()
 
 describe('spot check unit test policy functions with SimplePolicy', () => {
   describe('Policy Client Tests', () => {
     beforeAll(async () => {
+      process.env.KAFKA_HOST = 'localhost:7375'
```

Comment:
Why 7375?

Reply:
I think because that's just the default kafka port